content/api/events.apib

```diff
@@ -0,0 +1,269 @@
+        + Default: `24 hours ago`
+    + to (string, optional) - Datetime in format of YYYY-MM-DDTHH:MM.
+        + Default: `1 minute ago`
+    + timezone (string, optional) - Standard timezone identification string. Used for both querying, as well as formatting for `timestamp` and `injection_time` fields in reults.
+        + Default: `UTC`
+    + cursor (string, optional) - Results cursor for pagination. Used in conjunction with `per_page` parameter. See [Pagination](/api/events/#events-pagination) section for details.
+        + Default: `initial`
+    + per_page (number, optional) - Maximum number of results to return per page. Must be between 1 and 10,000.
+        + Default: `1000`
+    + delimiter (optional, string) ... Specifies the delimiter for query parameter lists.
+        + Default: `,`
+    + events (string, optional) - Comma delimited list of event types to search.
+        + Default: `all event types`
+    + recipients (string, optional, `recipient@example.com`) ... Comma delimited list of recipients to search.
+    + recipient_domains (string, optional) ... Comma delimited list of recipient domains to search. Supports wildcarding.
+    + from_addresses (string, optional) - Comma delimited list of friendly from addresses to search.
+    + sending_domains (string, optional) - Comma delimited list of sending domains to search. Supports wildcarding.
+    + subject (string, optional) - Subject line from the email header to search. Supports wildcarding.
+    + bounce_classes (number, optional) - Comma delimited list of bounce classification codes to search. See [Bounce Classification Codes](https://www.sparkpost.com/docs/deliverability/bounce-classification-codes/).
+    + reason (optional, string) ... Bounce/failure/rejection reason to search. Supports wildcarding.
+    + campaigns (string, optional) ... Comma delimited list of campaign IDs to search. Supports wildcarding.
+    + templates (string, optional, `my-template`) ... Comma delimited list of template IDs to search. Supports wildcarding.
+    + sending_ips (string, optional) - Comma delimited list of sending IP addresses to search.
+    + ip_pools (string, optional) - Comma delimited list of IP pool IDs to search. Supports wildarding.
+    + subaccounts (string, optional) - Comma delimited list of subaccount IDs to search.
+    + message_ids (string, optional) - Comma delimited list of message IDs to search.
+    + transmission_ids (string, optional) - Comma delimited list of transmission IDs to search.
+    + ab_tests (string, optional) - Comma delimited list of A/B test IDs to search. Supports wildcarding.
+    + ab_test_version (number, optional) - Version number of A/B test to search. If provided, `ab_tests` parameter becomes required.
+
++ Request
+
+    + Headers
+
+        Accept: application/json
+        Authorization: 14ac5499cfdd2bb2859e4476d2e5b1d2bad079bf
+
++ Response
+    
+        {
+            "results": [
```

Comment:
sample response can be updated with `recipient_domain` and `sending_domain` fields